Lesson4/Helpers/GenerateUser.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  GenerateUser.swift
+//  Lesson4
+//
+//  Created by Милана Махсотова on 15.10.2021.
+//
+
+import Foundation
+import UIKit
+
+class GenerateUser {
```

Comment:
```suggestion
class UserGenerator {
```
Не может класс называться как действие6 действие относится к функциям